tests/Feature/ReviewTableViewTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace LaravelViews\Test\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use LaravelViews\Test\Database\FoodTest;
+use LaravelViews\Test\Database\ReviewTest;
+use LaravelViews\Test\Database\UserTest;
+use LaravelViews\Test\Mock\MockReviewTableView;
+use LaravelViews\Test\Mock\MockReviewTableViewWithSearch;
+use LaravelViews\Test\Mock\MockTableViewWithSearchAndFilters;
+use LaravelViews\Test\TestCase;
+use Livewire\Livewire;
+
+class ReviewTableViewTest extends TestCase
+{
+    use RefreshDatabase;
+
+
+    private $livewire;
+    private $users;
+    private $reviews;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        factory(FoodTest::class, 10)->create();
+        $this->users = factory(UserTest::class, 10)
+            ->create()
+            ->each(function (UserTest $user) {
+                $user->reviews()->saveMany(factory(ReviewTest::class, 1)->make());
```

Comment:
I think you don't need to specify the `1` to create just one object, without that argument should work out as well

Reply:
I actually had 10 there and had some issues when trying to assert while we have pagination. I will remove it.